internal/webapi/getfeeaddress.go

```diff
@@ -62,7 +62,7 @@ func (s *server) getCurrentFee(dcrdClient *rpc.DcrdRPC) (dcrutil.Amount, error)
 	}
 
 	fee := txrules.StakePoolTicketFee(sDiff, defaultMinRelayTxFee,
-		int32(bestBlock.Height), s.cfg.VSPFee, s.cfg.NetParams, isDCP0010Active)
+		int32(bestBlock.Height), s.cfg.VSPFee, s.cfg.Network.Params, isDCP0010Active)
```

Comment:
Not sure how strict you are in vspd repo, but this and several others are > 80 now.  I see others that were already that way too, hence I'll avoid further commenting on it.

Reply:
I'm not typically too strict about it because I feel in some cases it can harm readability more than it helps. I really don't like it when a line of code becomes a paragraph. So long as its not unreasonable, like 150, I let it slide.